config/docker/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:xenial
+
+ARG PLASO_REVISION
+
+######################
+# Install basic deps #
+######################
+RUN apt-get clean
+RUN apt-get update
+RUN apt-get -y upgrade
+RUN apt-get install -y build-essential \
+  autotools-dev \
+  libsqlite3-dev \
+  python-dev \
+  debhelper \
+  devscripts \
+  fakeroot \
+  quilt \
+  git \
+  mercurial \
+  python-setuptools \
+  libtool \
+  automake \
+  libyaml-dev \
+  software-properties-common \
+  wget \
+  sudo
+
+####################
+# Add GIFT dev PPA #
+####################
+RUN add-apt-repository ppa:gift/dev
+RUN mkdir install
+WORKDIR install
+RUN wget https://raw.githubusercontent.com/log2timeline/plaso/${PLASO_REVISION}/config/linux/gift_ppa_install.sh
```

Comment:
Shouldn't this just run gift_ppa_install.sh from the mapped in Plaso source tree?

Reply:
Yep I originally did it this way because PLASO_SRC isn't in the Docker build context, but I think I can specify the build context so I can get rid of this